cmake/test.cmake

```diff
@@ -191,6 +191,12 @@ finally:
     list(APPEND mayaUsd_varname_PXR_PLUGINPATH_NAME
          "${CMAKE_INSTALL_PREFIX}/plugin/al/plugin")
 
+    if(IS_WINDOWS AND DEFINED ENV{PYTHONHOME})
+        # If the environment contains a PYTHONHOME, also set the path to
+        # that folder so that we can find the python DLLs.
+        list(APPEND mayaUsd_varname_PATH $ENV{PYTHONHOME})
+    endif()
```

Comment:
@HamedSabri-adsk As we discussed, when the PYTHONHOME env var is set (which we'll do from our internal builds) we also add that to the path. This is so the tests can find the python37.dll